.env.test.integration

```diff
@@ -1,3 +1,5 @@
+export PW_EXPERIMENTAL_TS_ESM=1
+export PW_MODE=integration
```

Comment:
Do we need `export`? Should we use it consistently?

Reply:
I think if we use dotenv, i.e. `require('dotenv').config()` we don't, but if we `. ./.env` from the command line we do. Maybe I'll just use `dotenv` in my scripts and nuke the exports